decoder/attribute_candidates.go

```diff
@@ -28,23 +28,24 @@ func attributeSchemaToCandidate(name string, attr *schema.AttributeSchema, rng h
 }
 
 func detailForAttribute(attr *schema.AttributeSchema) string {
-	var detail string
+	details := []string{}
+
 	if attr.IsRequired {
-		detail = "Required"
+		details = append(details, "required")
```

Comment:
This is a reasonable change, but just FYI there will be some existing test data in the need of update potentially in `terraform-schema` and in `terraform-ls` 😅

Reply:
sounds exciting  🙈